internal/proxy/task.go

```diff
@@ -1337,6 +1337,23 @@ func (st *SearchTask) getVChannels() ([]vChan, error) {
 	return st.chMgr.getVChannels(collID)
 }
 
+// https://github.com/milvus-io/milvus/issues/6411
+// Support wildcard match
+func translateOutputFields(outputFields []string, schema *schemapb.CollectionSchema) ([]string, error) {
```

Comment:
Please do a trim operation on all strings, so that "  *   " and  "*" should have the same meaning

Reply:
OK, I will fix this soon.

Reply:
Fixed and related test cases was also added.